src/Diagnostics.DataProviders/DataProviderConfigurations/GeoMasterDataProviderConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace Diagnostics.DataProviders
+{
+    [DataSourceConfiguration("GeoMaster")]
+    public class GeoMasterDataProviderConfiguration : IDataProviderConfiguration
+    {
+        
+        public void PostInitialize()
+        {
+        }
+
+        public GeoMasterDataProviderConfiguration()
+        {
+        }
+
+        /// <summary>
+        /// GeomasterCertThumbprint
+        /// </summary>
+        [ConfigurationName("GeoRegionCertThumbprint")]
+        public string GeoRegionCertThumbprint { get; set; }
```

Comment:
nitpik: Could you name this GeoMasterProdCertThumbprint instead. This name gives me the impression that you are trying to authenticate with GR instead of GeoMaster.

Reply:
Actually this comment is interesting. I am just trying to be in exact sync with the RoleConfiguration setting for diagrole so it is easy to relate what is what. Earlier I had it named as GeoMasterCertThumbprint and I was reading this setting from roleconfiguration but then I ended reading the totally wrong certificate and wasted many hours. I am not sure why role configuration has it named this way but I am thinking changing the name now will cause more confusion? Or not?

Reply:
If you look in Microsoft.Web.Hosting.Diagnstics.Common CommonRuntimeConfiguration.cs you will see this:

![image](https://user-images.githubusercontent.com/7739017/38886330-7c4bdaa2-422a-11e8-8166-7cb6b2be2854.png)

Reply:
Ouch...that's a bigger problem then. I will validate and update everywhere

Reply:
Changed from GeoRegionCertThumbprint to GeoCertThumbprint